pyproject.toml

```diff
@@ -20,15 +20,23 @@ keywords = ["invenio", "edugain"]
 license = {file = "LICENSE"}
 readme = "README.md"
 requires-python = ">=3.9"
-dependencies = []
+dependencies = [
+  "invenio_db>=2.0.0,<3.0.0",
+  "pysaml2>=7.5.0",
+]
 
 [project.optional-dependencies]
 tests = [
+  "invenio-app>=2.0.0,<3.0.0",
+  "invenio-db[mysql,postgresql]>=2.0.0,<3.0.0",
```

Comment:
please remove mysql here, we only test for postgresql

Reply:
The inclusion of `[mysql]` extra in tests for invenio core modules confused me here.
But CI workflow's testing-matrix doesn't test for `mysql` anyway... (see [here](https://github.com/inveniosoftware/workflows/blob/master/.github/workflows/tests-python.yml))

Removed.